File: Beginner/heavenEarth/heavenEarth.c
#include <stdio.h>
#include <assert.h>
#include <math.h>

int main(void){
	int T,N;
	double V1,V2;

	scanf("%d",&T);
	assert(T>=1 && T<=1000);

	while(T>0){
		scanf("%d %lf %lf",&N,&V1,&V2);
		assert(N>=1 && N<=100);
		assert(V1>=1 && V1<=100);
		assert(V2>=1 && V2<=100);

		if(V2<sqrt(2)*V1){
			printf("Stairs\n");
		}else{
			printf("Elevator\n");
		}

		T--;
	}


	return 0;
}

File: Beginner/heavenEarth/heavenEarth.java
//run as java -ea heavenEarth
import java.util.Scanner;
import java.lang.Math;

public class heavenEarth{
	public static void main(String[] args){
		int T,N;
		float V1,V2;
		Scanner scan = new Scanner(System.in);

		T = scan.nextInt();
		assert(T>=1 && T<=1000) : "T must be 1 <= T <= 1000";

		while(T>0){
			N = scan.nextInt();
			V1 = scan.nextFloat();
			V2 = scan.nextFloat();

			assert(N>=1 && N<=100) : "N must be 1 <= N <= 100";
			assert(V1>=1 && V1<=100) : "V1 must be 1 <= V1 <= 100";
			assert(V2>=1 && V2<=100) : "V2 must be 1 <= V2 <= 100";

			if(V2<Math.sqrt(2)*V1){
				System.out.println("Stairs");
			}else{
				System.out.println("Elevator");
			}

			T--;
		}	
	}
}